v2v-cli/cmd/generate.go

```diff
@@ -143,6 +143,20 @@ Your admin.rc file should atleast contain the following keys: OS_AUTH_URL, OS_DO
 						value = "false"
 					}
 				}
+			case "CONVERT":
+				if convert, _ := cmd.Flags().GetString("convert"); convert != "" {
+					value = convert
+				} else {
+					fmt.Printf("Enter value for %s (true/false) (Default is true):", env)
+					fmt.Scanln(&value)
+					if value == "" {
+						value = "true"
+					} else if strings.ToLower(value) == "true" || strings.ToLower(value) == "t" {
+						value = "true"
+					} else if strings.ToLower(value) == "false" || strings.ToLower(value) == "f" {
+						value = "false"
+					}
```

Comment:
create a function to read boolean value instead of embedding it here

Reply:
fixed